core/src/test/java/ai/timefold/solver/core/impl/score/stream/common/bi/AbstractBiConstraintStreamTest.java

```diff
@@ -942,16 +942,16 @@ public void groupBy_1Mapping1Collector() {
         // From scratch
         scoreDirector.setWorkingSolution(solution);
         assertScore(scoreDirector,
-                assertMatchWithScore(-1, TEST_CONSTRAINT_NAME, solution.getFirstEntity().toString(), 6),
-                assertMatchWithScore(-1, TEST_CONSTRAINT_NAME, solution.getEntityList().get(1).toString(), 5));
+                assertMatchWithScore(-1, null, TEST_CONSTRAINT_NAME, solution.getFirstEntity().toString(), 6),
```

Comment:
Are we still using the constraint package elsewhere? If not, why not remove it instead of using a `null` value?

Reply:
I needed to be able to test what happens when there is no package. 
Which means I need to have a way to provide the package. (Which, until now, didn't exist in the test.)